src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporter.cs

```diff
@@ -90,10 +90,10 @@ public override ExportResult Export(in Batch<LogRecord> logRecordBatch)
         // Prevents the exporter's gRPC and HTTP operations from being instrumented.
         using var scope = SuppressInstrumentationScope.Begin();
 
+        var request = this.otlpLogRecordTransformer.BuildExportRequest(this.ProcessResource, logRecordBatch);
```

Comment:
Why move this outside of the `try` block?

Reply:
Not intentional. Fixed it